geomstats/backend/numpy.py

```diff
@@ -290,3 +290,15 @@ def nonzero(x):
 
 def copy(x):
     return np.copy(x)
+
+
+def sign(*args, **kwargs):
+    return np.sign(*args, **kwargs)
+
+
+def mean(x, axis=None):
+    return np.mean(x, axis)
+
+
+def normal(*args, **kwargs):
```

Comment:
Put in numpy_random.py

Reply:
Done, along with adding normal() to pytorch_random.py and tensorflow_random.py